### DISCUSSION TITLE
What is the best way to define custom metadata for tabular resources?

### DISCUSSION THREAD

**Discussion by augusto-herrmann** [frictionlessdata/frictionlessdata.io#605] [Q&A]
When I need to define custom metadata for a resource, I could use [Profiles](https://specs.frictionlessdata.io/profiles/) and define a custom JSON Schema profile for the `Resource`. However, since it's no longer a [Tabular Data Resource](https://specs.frictionlessdata.io/tabular-data-resource/), tools like [Good Tables](https://github.com/frictionlessdata/goodtables-py) will no longer validate field types in columns and other restrictions on data that are so useful to automate.

Is there a way to define custom metadata for tabular resources, while still maintaining tool compatibility with tools that handle Table Schemas?

**Comment by roll** [comment]
Thatks @augusto-herrmann,

It's a good question to start a discussion. I'm cc @rufuspollock
